### ISSUE TITLE
libjava2cpython.so: undefined symbol: Py_IsInitialized

### ISSUE THREAD

**Issue by i-akbari** [LeeKamentsky/python-javabridge#110]
I installed javabridge but when I want to instantiate CPython it gives me error:

Minimal Code: 
Note that the code is built without error.
```
package callpython;

import org.cellprofiler.javabridge.CPython;

class CallPython {
    public static void main(String[] args) {
      CPython cpython = new CPython();       
    }
}
```

Runtime Error:
> /usr/local/java/jdk1.8.0_20/bin/java: symbol lookup error: /usr/local/lib/python3.4/dist-packages/javabridge/jars/libjava2cpython.so: undefined symbol: Py_IsInitialized
> /home/.../.cache/netbeans/8.1/executor-snippets/run.xml:53: Java returned: 127
> BUILD FAILED (total time: 0 seconds)

**Comment by i-akbari** [comment]
Maybe this is helpful to detect the problem.

ldd -r _javabridge.cpython-34m.so

> 	linux-vdso.so.1 =>  (0x00007ffc07fdc000)
> 	/usr/lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007f7cf9848000)
> 	libjvm.so => /usr/lib/libjvm.so (0x00007f7cf88c6000)
> 	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f7cf86a8000)
> 	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f7cf82e3000)
> 	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f7cf7fdd000)
> 	/lib64/ld-linux-x86-64.so.2 (0x00007f7cf9d96000)
> 	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f7cf7dc7000)
> 	libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f7cf7bc3000)
> undefined symbol: PyExc_SystemError	(./_javabridge.cpython-34m.so)
> undefined symbol: PyMethod_Type	(./_javabridge.cpython-34m.so)
> undefined symbol: PyFloat_Type	(./_javabridge.cpython-34m.so)
> undefined symbol: PyTuple_Type	(./_javabridge.cpython-34m.so)
> undefined symbol: PyNumber_InPlaceAdd	(./_javabridge.cpython-34m.so)
> undefined symbol: PyNumber_Add	(./_javabridge.cpython-34m.so)
> undefined symbol: PyByteArray_Type	(./_javabridge.cpython-34m.so)
> undefined symbol: PyBytes_Type	(./_javabridge.cpython-34m.so)
> undefined symbol: _PyByteArray_empty_string	(./_javabridge.cpython-34m.so)
> undefined symbol: PyFPE_jbuf	(./_javabridge.cpython-34m.so)
> undefined symbol: PyExc_RuntimeError	(./_javabridge.cpython-34m.so)
> undefined symbol: PyExc_ValueError	(./_javabridge.cpython-34m.so)
> undefined symbol: PyExc_TypeError	(./_javabridge.cpython-34m.so)
> undefined symbol: _PyThreadState_Current	(./_javabridge.cpython-34m.so)
> undefined symbol: _Py_CheckRecursionLimit	(./_javabridge.cpython-34m.so)
> undefined symbol: _Py_FalseStruct	(./_javabridge.cpython-34m.so)
> undefined symbol: PyExc_OverflowError	(./_javabridge.cpython-34m.so)
> undefined symbol: PyExc_FloatingPointError	(./_javabridge.cpython-34m.so)
> undefined symbol: Py_OptimizeFlag	(./_javabridge.cpython-34m.so)
> undefined symbol: _Py_NoneStruct	(./_javabridge.cpython-34m.so)
> undefined symbol: PyExc_AssertionError	(./_javabridge.cpython-34m.so)
> undefined symbol: _Py_TrueStruct	(./_javabridge.cpython-34m.so)
> undefined symbol: PyBaseObject_Type	(./_javabridge.cpython-34m.so)
> undefined symbol: PyLong_Type	(./_javabridge.cpython-34m.so)
> undefined symbol: PyCapsule_Type	(./_javabridge.cpython-34m.so)
> undefined symbol: PyExc_ImportError	(./_javabridge.cpython-34m.so)
> undefined symbol: PyExc_AttributeError	(./_javabridge.cpython-34m.so)
> undefined symbol: PyExc_StopIteration	(./_javabridge.cpython-34m.so)
> undefined symbol: PyUnicode_Type	(./_javabridge.cpython-34m.so)
> undefined symbol: PyExc_NameError	(./_javabridge.cpython-34m.so)
> undefined symbol: PyFPE_counter	(./_javabridge.cpython-34m.so)
> undefined symbol: PyCFunction_Type	(./_javabridge.cpython-34m.so)
> undefined symbol: PyMethodDescr_Type	(./_javabridge.cpython-34m.so)
> undefined symbol: PySet_Type	(./_javabridge.cpython-34m.so)
> undefined symbol: PyList_Type	(./_javabridge.cpython-34m.so)
> undefined symbol: PyUnicode_FromFormat	(./_javabridge.cpython-34m.so)
> undefined symbol: PyList_New	(./_javabridge.cpython-34m.so)
> undefined symbol: _PyUnicode_Ready	(./_javabridge.cpython-34m.so)
> undefined symbol: PyDict_SetItemString	(./_javabridge.cpython-34m.so)
> undefined symbol: PyDict_Size	(./_javabridge.cpython-34m.so)
> undefined symbol: PyException_SetTraceback	(./_javabridge.cpython-34m.so)
> undefined symbol: PyErr_Restore	(./_javabridge.cpython-34m.so)
> undefined symbol: PyEval_RestoreThread	(./_javabridge.cpython-34m.so)
> undefined symbol: PyFrame_New	(./_javabridge.cpython-34m.so)
> undefined symbol: PyCFunction_NewEx	(./_javabridge.cpython-34m.so)
> undefined symbol: PyBuffer_Release	(./_javabridge.cpython-34m.so)
> undefined symbol: PyNumber_Add	(./_javabridge.cpython-34m.so)
> undefined symbol: PyObject_GetAttrString	(./_javabridge.cpython-34m.so)
> undefined symbol: PyImport_AddModule	(./_javabridge.cpython-34m.so)
> undefined symbol: PyBytes_FromStringAndSize	(./_javabridge.cpython-34m.so)
> undefined symbol: PyObject_SetAttrString	(./_javabridge.cpython-34m.so)
> undefined symbol: PyErr_WarnEx	(./_javabridge.cpython-34m.so)
> undefined symbol: PyErr_SetObject	(./_javabridge.cpython-34m.so)
> undefined symbol: PyErr_NormalizeException	(./_javabridge.cpython-34m.so)
> undefined symbol: PyLong_FromSize_t	(./_javabridge.cpython-34m.so)
> undefined symbol: PyObject_RichCompare	(./_javabridge.cpython-34m.so)
> undefined symbol: PyCode_New	(./_javabridge.cpython-34m.so)
> undefined symbol: PyImport_GetModuleDict	(./_javabridge.cpython-34m.so)
> undefined symbol: PyFPE_dummy	(./_javabridge.cpython-34m.so)
> undefined symbol: PyNumber_Long	(./_javabridge.cpython-34m.so)
> undefined symbol: PyErr_GivenExceptionMatches	(./_javabridge.cpython-34m.so)
> undefined symbol: PyErr_SetString	(./_javabridge.cpython-34m.so)
> undefined symbol: PyObject_GetItem	(./_javabridge.cpython-34m.so)
> undefined symbol: PyErr_Print	(./_javabridge.cpython-34m.so)
> undefined symbol: PyMem_Realloc	(./_javabridge.cpython-34m.so)
> undefined symbol: PyErr_SetNone	(./_javabridge.cpython-34m.so)
> undefined symbol: PyErr_ExceptionMatches	(./_javabridge.cpython-34m.so)
> undefined symbol: PyOS_snprintf	(./_javabridge.cpython-34m.so)
> undefined symbol: PyTraceBack_Here	(./_javabridge.cpython-34m.so)
> undefined symbol: PyObject_CallFinalizerFromDealloc	(./_javabridge.cpython-34m.so)
> undefined symbol: PyLong_FromSsize_t	(./_javabridge.cpython-34m.so)
> undefined symbol: PyFloat_FromDouble	(./_javabridge.cpython-34m.so)
> undefined symbol: PyType_Ready	(./_javabridge.cpython-34m.so)
> undefined symbol: PyLong_FromLong	(./_javabridge.cpython-34m.so)
> undefined symbol: PyLong_AsSsize_t	(./_javabridge.cpython-34m.so)
> undefined symbol: PyDict_GetItem	(./_javabridge.cpython-34m.so)
> undefined symbol: PyUnicode_FromUnicode	(./_javabridge.cpython-34m.so)
> undefined symbol: PyErr_Clear	(./_javabridge.cpython-34m.so)
> undefined symbol: PyList_Append	(./_javabridge.cpython-34m.so)
> undefined symbol: _Py_CheckRecursiveCall	(./_javabridge.cpython-34m.so)
> undefined symbol: PyUnicode_AsUnicode	(./_javabridge.cpython-34m.so)
> undefined symbol: PyTuple_New	(./_javabridge.cpython-34m.so)
> undefined symbol: PyObject_SetAttr	(./_javabridge.cpython-34m.so)
> undefined symbol: PyBytes_FromString	(./_javabridge.cpython-34m.so)
> undefined symbol: PyErr_Occurred	(./_javabridge.cpython-34m.so)
> undefined symbol: PyModule_Create2	(./_javabridge.cpython-34m.so)
> undefined symbol: PyLong_AsLong	(./_javabridge.cpython-34m.so)
> undefined symbol: PyImport_ImportModule	(./_javabridge.cpython-34m.so)
> undefined symbol: PyTuple_GetSlice	(./_javabridge.cpython-34m.so)
> undefined symbol: PyDict_GetItemString	(./_javabridge.cpython-34m.so)
> undefined symbol: PyObject_Size	(./_javabridge.cpython-34m.so)
> undefined symbol: PyUnicode_DecodeUTF8	(./_javabridge.cpython-34m.so)
> undefined symbol: PyFloat_AsDouble	(./_javabridge.cpython-34m.so)
> undefined symbol: PyObject_IsTrue	(./_javabridge.cpython-34m.so)
> undefined symbol: PyImport_ImportModuleLevelObject	(./_javabridge.cpython-34m.so)
> undefined symbol: PyUnicode_Compare	(./_javabridge.cpython-34m.so)
> undefined symbol: PyDict_New	(./_javabridge.cpython-34m.so)
> undefined symbol: PyObject_GetBuffer	(./_javabridge.cpython-34m.so)
> undefined symbol: PyDict_Next	(./_javabridge.cpython-34m.so)
> undefined symbol: PyErr_Fetch	(./_javabridge.cpython-34m.so)
> undefined symbol: PyUnicode_FromString	(./_javabridge.cpython-34m.so)
> undefined symbol: PyObject_GetIter	(./_javabridge.cpython-34m.so)
> undefined symbol: PyEval_SaveThread	(./_javabridge.cpython-34m.so)
> undefined symbol: PyUnicode_InternFromString	(./_javabridge.cpython-34m.so)
> undefined symbol: PyDict_SetItem	(./_javabridge.cpython-34m.so)
> undefined symbol: PyObject_HasAttr	(./_javabridge.cpython-34m.so)
> undefined symbol: PyUnicode_DecodeUTF16	(./_javabridge.cpython-34m.so)
> undefined symbol: PyObject_Call	(./_javabridge.cpython-34m.so)
> undefined symbol: PyType_IsSubtype	(./_javabridge.cpython-34m.so)
> undefined symbol: PyUnicode_Decode	(./_javabridge.cpython-34m.so)
> undefined symbol: PyErr_Format	(./_javabridge.cpython-34m.so)
> undefined symbol: PyCapsule_GetPointer	(./_javabridge.cpython-34m.so)
> undefined symbol: PySlice_New	(./_javabridge.cpython-34m.so)
> undefined symbol: PyUnicode_FromStringAndSize	(./_javabridge.cpython-34m.so)
> undefined symbol: PyModule_GetDict	(./_javabridge.cpython-34m.so)
> undefined symbol: PySet_Pop	(./_javabridge.cpython-34m.so)
> undefined symbol: PyNumber_Index	(./_javabridge.cpython-34m.so)
> undefined symbol: PyObject_GetAttr	(./_javabridge.cpython-34m.so)
> undefined symbol: PyBytes_AsStringAndSize	(./_javabridge.cpython-34m.so)
> undefined symbol: PyUnicode_Format	(./_javabridge.cpython-34m.so)
> undefined symbol: PyMem_Malloc	(./_javabridge.cpython-34m.so)
> undefined symbol: PyTuple_Pack	(./_javabridge.cpython-34m.so)
> undefined symbol: Py_GetVersion	(./_javabridge.cpython-34m.so)
> undefined symbol: PyObject_GC_UnTrack	(./_javabridge.cpython-34m.so)
> undefined symbol: PyErr_WriteUnraisable	(./_javabridge.cpython-34m.so)
> undefined symbol: PyImport_Import	(./_javabridge.cpython-34m.so)
> undefined symbol: PyFloat_FromString	(./_javabridge.cpython-34m.so)

ldd -r libjava2cpython.so
> 	linux-vdso.so.1 =>  (0x00007ffffd940000)
> 	/usr/lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007fab776eb000)
> 	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007fab774cd000)
> 	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007fab77108000)
> 	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007fab76e02000)
> 	/lib64/ld-linux-x86-64.so.2 (0x00007fab77bf3000)
> 	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007fab76bec000)
> undefined symbol: dlopen	(./libjava2cpython.so)
> undefined symbol: Py_Initialize	(./libjava2cpython.so)
> undefined symbol: PyGILState_Release	(./libjava2cpython.so)
> undefined symbol: PyObject_CallMethod	(./libjava2cpython.so)
> undefined symbol: PyObject_GetAttrString	(./libjava2cpython.so)
> undefined symbol: PyImport_AddModule	(./libjava2cpython.so)
> undefined symbol: PyDict_Merge	(./libjava2cpython.so)
> undefined symbol: PyGILState_Ensure	(./libjava2cpython.so)
> undefined symbol: PyErr_Print	(./libjava2cpython.so)
> undefined symbol: PyErr_Clear	(./libjava2cpython.so)
> undefined symbol: PyTuple_New	(./libjava2cpython.so)
> undefined symbol: PyErr_Occurred	(./libjava2cpython.so)
> undefined symbol: PyImport_ImportModule	(./libjava2cpython.so)
> undefined symbol: PyString_AsString	(./libjava2cpython.so)
> undefined symbol: PyObject_CallObject	(./libjava2cpython.so)
> undefined symbol: PyRun_StringFlags	(./libjava2cpython.so)
> undefined symbol: Py_IsInitialized	(./libjava2cpython.so)
> undefined symbol: PyDict_New	(./libjava2cpython.so)
> undefined symbol: PyCapsule_New	(./libjava2cpython.so)
> undefined symbol: PyModule_GetDict	(./libjava2cpython.so)
> undefined symbol: PyTuple_Pack	(./libjava2cpython.so)

**Comment by i-akbari** [comment]
This is the same problem as "https://github.com/LeeKamentsky/python-javabridge/issues/102" that has answered. 

It seems that the fixed version 1.0.14 is not working for me but the solution is:

System.setProperty("python.location", "/usr/lib/python3.4/config-3.4m-x86_64-linux-gnu/libpython3.4.so");
